v2g-liberty-cards/src/car-reservation-calendar-settings-card.ts

```diff
@@ -90,6 +96,28 @@ export class CarReservationCalendarSettingsCard extends LitElement {
       : nothing;
   }
 
+
+  private _renderCaldavConnectionStatus() {
+    const state = this._caldavConnectionStatus.state;
+    const isConnected = state === 'Successfully connected';
```

Comment:
In other places I created some enums for the string constants. We could consider doing the same here.